doc/manual/manual.tex

```diff
@@ -9681,6 +9681,69 @@ \subsubsection{Thin shell gravity benchmark}
+\end{equation}
+The solution to this expression is:
+\begin{equation}
+g=\frac{\partial U}{\partial r} = \frac{C}{r^2} \label{eq:app1},
+\end{equation}
+where $C$ is a constant of integration.
+In order to avoid an infinite gravity field at $r=0$ (where the density is also zero in this particular setup of a shell), we need to impose $C=0$, i.e. the gravity is zero for $r\leq R\textsubscript{inner}$. Another way of arriving at the same conclusion is to realize that $g$ is zero at the centre of the body because the material around it exerts an equal force in every direction.
+Inside the shell, $\rho=\rho_0$, yielding
+\begin{equation}
+g=\frac{\partial U}{\partial r} = \frac{4 \pi}{3} G \rho_0 r + \frac{A}{r^2},
+\end{equation}
+where $A$ is another integration constant. 
+At the inner boundary, $r=R\textsubscript{inner}$ and $g=0$, allowing $A$ to be computed. Substituting in the value of $A$,
+\begin{equation}
+g=\frac{\partial U}{\partial r} = \frac{4 \pi}{3} G \rho_0 
+\left(r - \frac{R\textsubscript{inner}^3}{r^2} \right). \label{eqgin}
+\end{equation}
+When $r\geq R\textsubscript{outer}$, the gravitational potential is given by Eq. (\ref{eq:app1}). Requiring the gravity field to be continuous at $r=R\textsubscript{outer}$:
+\begin{equation}
+g(r) = \frac{G M}{r^2} \label{eqgout},
+\end{equation}
+where $M=\frac{4 \pi}{3} \rho_0(R\textsubscript{outer}^3-R\textsubscript{inner}^3)$ is the mass contained in the shell.
+For $r\ge R\textsubscript{outer}$, the potential is obtained by integrating Eq.(\ref{eqgout}):
+\begin{equation}
+U(r)=-\frac{GM}{r} +D,
+\end{equation}
+where $D$ is an integration constant which has to be zero since we require the potential to vanish for $r\rightarrow \infty$.
+The potential within the shell, $R\textsubscript{inner}\leq r \leq R\textsubscript{outer}$, is found by integrating Eq.~\eqref{eqgin}:
+\begin{equation}
+U(r)= \frac{4 \pi}{3} G \rho_0 \left(\frac{r^2}{2} + \frac{R\textsubscript{inner}^3}{r} \right)  + E,
+\end{equation}
+where $E$ is a constant. Continuity of the potential at $r=R\textsubscript{outer}$ requires that $E=-2\pi\rho_0 G R\textsubscript{outer}^2$. 
+Gravitational acceleration is zero for $r\leq R\textsubscript{inner}$, so the potential there is constant and a continuity requirement yields
+\begin{equation}
+U(r)=2\pi G \rho_0 (R\textsubscript{inner}^2-R\textsubscript{outer}^2).
+\end{equation}
+
+The gravity postprocessor in \aspect{} can be used to calculate the radial components of gravity ($g_r$ and $U$) at arbitrary points using the Sampling scheme `\textit{list of points}'. For this benchmark we calculate points along a line from the center of the planet to a distant point, $r=0$ to $r=10,000~\si{\km}$ (Figure \ref{fig:grav-thick-shell}). Arbitrarily, the latitude and longitude are both set to $13\si{\degree}$ so as to avoid potential measurement artefacts due to symmetry. The list of radii is defined as follows:
```

Comment:
```suggestion
The gravity postprocessor in \aspect{} can be used to calculate the radial components of gravity ($g_r$ and $U$) at arbitrary points using the sampling scheme `\textit{list of points}'. For this benchmark we calculate points along a line from the center of the planet to a distant point, $r=0$ to $r=10,000~\si{\km}$ (Figure \ref{fig:grav-thick-shell}). Arbitrarily, the latitude and longitude are both set to $13\si{\degree}$ so as to avoid potential measurement artifacts due to symmetry. The list of radii is defined as follows:
```

Reply:
well, I learned something. Apparently both spellings are ok: https://en.wikipedia.org/wiki/Artifact_(error)  :)

Reply:
Yes, I think artefact is British English and artifact American English: https://en.wiktionary.org/wiki/artifact